config/eslintrc_react.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
         'react/no-is-mounted': 2,
         'react/no-multi-comp': 0, // Enable to define a multiple component to a single file.
         'react/no-render-return-value': 2,
+        'react/no-redundant-should-component-update': 0, // we'll reconsider this option when we begin to use `PureComponent`.
```

Comment:
In reality, I don't understand the benefit of `ignorePureComponents` yet, because I'd like to use stateless component as possible as we can.

However I don't have any strong opinion about the option. I'm okay to enable it.